webapp-django/crashstats/crashstats/models.py

```diff
@@ -896,8 +929,31 @@ class SignaturesByBugs(SocorroMiddleware):
         ),
     }
 
+    def get(self, *args, **kwargs):
```

Comment:
This returns the signatures for a set of bugs. Because of that, we just use `.filter()`.